source/aggregation-tutorials.txt

```diff
@@ -1,12 +1,104 @@
+.. _node-aggregation-tutorials-landing:
+
 =====================
 Aggregation Tutorials
 =====================
 
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: node.js, code example, runnable app
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
 .. toctree::
 
    /aggregation-tutorials/one-to-one-join.txt
 
-Read the following tutorials to learn how perform specific tasks using
-aggregation:
+Overview
+--------
+
+Aggregation tutorials provide detailed explanations of common
+aggregation tasks in a step-by-step format. The tutorials are adapted
+from examples in the `Practical MongoDB Aggregations book
+<https://www.practical-mongodb-aggregations.com/>`__ by Paul Done.
+
+Each tutorial includes the following sections:
+
+- **Introduction**, which describes the purpose and common use cases of the
+  aggregation type. This section also describes the example and desired
+  outcome that the tutorial demonstrates.
+
+- **Before You Get Started**, which describes the necessary databases,
+  collections, and sample data that you must insert before building the
+  aggregation pipeline and performing the aggregation.
+
+- **Tutorial**, which describes how to build and run the aggregation
+  pipeline. This section describes each stage of the completed
+  aggregation tutorial, and then explains how to run and interpret the
+  output of the aggregation.
+
+At the end of each aggregation tutorial, you can find a link to a fully
+runnable Node.js code file that you can run in your environment.
+
+.. tip::
+
+   To learn more about performing aggregations, see the
+   :ref:`node-aggregation` guide.
+
+.. _node-agg-tutorial-template-app:
+
+Aggregation Template App
+------------------------
+
+Before you begin following an aggregation tutorial, set up an app that
+you can use to insert sample data into MongoDB and run the aggregation
+pipeline.
+
+.. tip:: 
+   
+   To learn how to install the driver and connect to MongoDB,
+   see the :ref:`node-quick-start-download-and-install` and
+   :ref:`node-quick-start-create-deployment` steps of the
+   Quick Start guide.
+
+Once you install the driver, create a file called
+``agg_tutorial.js``. Paste the following code in this file to create an
+app template for the aggregation tutorials:
+
+.. literalinclude:: /includes/aggregation/template-app.js
+   :language: javascript
+   :copyable: true
+
+.. important::
+
+   In the preceding code, read the code comments to find the sections of
+   the code that you must modify for the tutorial you are following.
+
+For every tutorial, you must replace the connection string placeholder with
+your deployment's connection string. For example, if your connection string is
+``"mongodb+srv://mongodb-example:27017"``, your connection string assignment resembles
+the following:
```

Comment:
```suggestion
For every tutorial, you must replace the connection string placeholder with
your deployment's driver connection string. For example, if your connection string is
``"mongodb+srv://mongodb-example:27017"``, your connection string assignment resembles
the following:
```
You may also want to note that the users IP addresses should be added to the Access List.

Reply:
What do you mean by access list?

Reply:
Also, I don't think connection strings are usually driver specific, so I don't think we should describe it as a driver connection string.

Reply:
>  I don't think connection strings are usually driver specific

They're not specific to an individual driver, but from my understanding, there are different types of connection strings for different types of access. So in this case, you want the string intended for drivers. This is the screen I'm thinking of, and where I got `driver` connection string from:
<img width="789" alt="Screenshot 2023-11-20 at 12 14 47 PM" src="https://github.com/mongodb/docs-node/assets/148898879/d16f404c-0ab6-48d7-b224-92e3db734ed8">

>What do you mean by access list?

I mean adding the IP address you'll be connecting from, to this list. I might be wrong tho, maybe this isn't required?

<img width="1695" alt="Screenshot 2023-11-20 at 12 18 55 PM" src="https://github.com/mongodb/docs-node/assets/148898879/2904d56d-0d2f-4ec7-9cad-c33475e08801">

Reply:
I believe configuring the access list is optional. Also, I think the Atlas connect modal just has the "drivers" option for connecting to show you additional steps to install the driver and the connection code -- even so, the connection string is not necessarily unique to drivers. I have used my connection string to connect by using a driver, but also for connecting within Compass and by using the shell.